.nvmrc

```diff
@@ -1 +1 @@
-node-version: 16.15.1
+node-version: v16.15.1
```

Comment:
remove the `node-version:` part

Reply:
done @holgiosalos

Reply:
I still see it. The content of this file should ve only:
```
v16.15.1
```

Reply:
done @holgiosalos